src/patterns/SaleHeaderBanner/_saleHeaderBanner.scss

```diff
@@ -61,7 +62,6 @@
       }
 
       &:first-child {
-        margin-bottom: 1.5rem;
```

Comment:
Not sure we do not have any stories with the button for the sale header banner but I think this may affect the spacing between the button and above content.

Reply:
Ok I figured out the spacing issue, we still need the 1.5rem bottom-margin, but it should be moved up a level to the `&-text` class instead of applying to first-child. I think that was leftover from when all the CTAs were controlled by this component.